skopt/searchcv.py

```diff
@@ -551,9 +552,33 @@ def _step(self, X, y, search_space, optimizer, groups=None, n_jobs=1):
         local_results = self.cv_results_['mean_test_score'][-len(params):]
 
         # optimizer minimizes objective, hence provide negative score
-        optimizer.tell(params, [-score for score in local_results])
+        return optimizer.tell(params, [-score for score in local_results])
 
-    def fit(self, X, y=None, groups=None):
+    @property
+    def total_iterations(self):
```

Comment:
This has zero coverage in any test, should be easy to check it works (and will keep working).

Reply:
Oops yes should have some tests :)